tests/common.go

```diff
@@ -2803,7 +2803,7 @@ func DeleteBackupLocation(name string, backupLocationUID string, orgID string, D
 }
 
 // DeleteSchedule deletes backup schedule
-func DeleteSchedule(backupScheduleName, backupScheduleUID, schedulePolicyName, schedulePolicyUID, OrgID string) error {
```

Comment:
This can be taken up in another PR. The clusterName should be passed to DeleteSchedule instead of assuming that it is always SourceClusterName.

Reply:
Ok, will take this in separate PR